lib/tests/streamlit/streamlit_test.py

```diff
@@ -60,13 +60,22 @@ def test_get_option(self):
         # This is set in lib/tests/conftest.py to False
         self.assertEqual(False, st.get_option("browser.gatherUsageStats"))
 
-    def test_set_option(self):
-        """Test streamlit.set_option."""
+    def test_set_option_scriptable(self):
+        """Test that scriptable options can be set from API."""
         # This is set in lib/tests/conftest.py to off
-        self.assertEqual("off", st.get_option("global.sharingMode"))
+        self.assertEqual(True, st.get_option("client.displayEnabled"))
 
-        st.set_option("global.sharingMode", "s3")
-        self.assertEqual("s3", st.get_option("global.sharingMode"))
+        # client.displayEnabled and client.caching can be set after run starts.
+        st.set_option("client.displayEnabled", False)
+        self.assertEqual(False, st.get_option("client.displayEnabled"))
```

Comment:
Nit, this tests is about _scriptable_ but requires that some specific options are "scriptable" and others are not.  The test may break if we change these options or just we change just their name. In my mind, it would be nice to create a fake "scriptable" and a non "scriptable" option and testing the fakes, but not sure if it is easy.

Reply:
Fair point!